system-filepath/lib/Filesystem/Path.hs

```diff
@@ -71,7 +71,7 @@ instance Sem.Semigroup FilePath where
 
 instance M.Monoid FilePath where
   mempty = empty
-  mappend = append
+  mappend = (<>)
```

Comment:
On GHCs 8.0 and 8.2 this causes the error
> Variable not in scope: (<>) :: FilePath -> FilePath -> FilePath

So let's wait with this until we drop support for these GHC versions.